libs/pages/services/src/lib/feature/page-helm-create-feature/step-summary-feature/step-summary-feature.tsx

```diff
@@ -287,11 +305,11 @@ export function StepSummaryFeature() {
               variant="surface"
               color="neutral"
               onClick={() => onSubmit(false)}
-              loading={isLoadingCreateHelm}
+              loading={isLoadingCreate}
             >
               Create
             </Button>
-            <Button type="submit" size="lg" onClick={() => onSubmit(true)} loading={isLoadingCreateHelm}>
+            <Button type="submit" size="lg" onClick={() => onSubmit(true)} loading={isLoadingCreateAndDeploy}>
```

Comment:
```suggestion
            <Button type="submit" size="lg" onClick={() => onSubmit(true)} loading={isLoadingCreateHelm || isLoadingDeployHelm}>
```
then you add `isLoading: isLoadingDeployHelm` in your `useDeployService` declaration

Reply:
We can't have this because we need two distincts loader for the two buttons, I added a disabled state when one is submit to avoid multiple requests